source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -687,8 +687,10 @@ static BaseType_t prvGMACInit( NetworkInterface_t * pxInterface )
     GMAC->GMAC_NCR |= GMAC_NCR_MPE;
 
     memset( &gmac_option, '\0', sizeof( gmac_option ) );
-    gmac_option.uc_copy_all_frame = 0;
-    gmac_option.uc_no_boardcast = 0;
+
+    /* Note that 'gmac_option.uc_copy_all_frame' is false, do not copy all frames.
+     * And 'gmac_option.uc_no_boardcast' is false, meaning that broadcast is received.
+     * 'boardcast' is a typo. */
```

Comment:
We can update it to uc_no_broadcast

Reply:
@moninom1 wrote:
> We can correct the name to be uc_no_broadcast

We can, but this is code provided by Microchip.

I had never noticed the typo `boardcast` until I noticed that broadcasts were not received. There is another misspelling `b_boardcast_detect`.

Reply:
Oh okay. Thanks for the info.